go.mod

```diff
@@ -28,5 +28,6 @@ require (
 
 replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v13.3.2+incompatible // Required by OLM
+	golang.org/x/tools => golang.org/x/tools v0.0.0-20200713011307-fd294ab11aed
```

Comment:
Ah, that explains why I was still seeing vendor issues in master.

Reply:
figure out today, the lib is updated almost daily so we need to pin until we figure out how to fix that and not have as hard dependency